### PR TITLE
Step 1 in displaying where users visit the site from: getting a map to show up

### PR DESCRIPTION
**Pull request by apluscs**

### CODE REVIEW

File: portfolio/src/main/webapp/map.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Creates a map and adds it to the page. */
+function createMap() {
+  console.log("map createing")
```
Comment (acmcarther) [inline portfolio/src/main/webapp/map.js:17]: Drop debug log?

File: portfolio/src/main/webapp/map.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  const map = new google.maps.Map(
+    document.getElementById('map'),
+    {center: {lat: 38.5, lng: -98}, zoom: 3}
```
Comment (acmcarther) [inline portfolio/src/main/webapp/map.js:20]: Nit: Make these coords a constant (or return them from a function with a good name).

File: portfolio/src/main/webapp/map.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Creates a map and adds it to the page. */
+function createMap() {
+  console.log("map createing")
+  const map = new google.maps.Map(
```
Comment (acmcarther) [inline portfolio/src/main/webapp/map.js:18]: Do you need to return this or store it or something?

File: portfolio/src/main/webapp/map.js (hunk 1 above)
Comment (apluscs) [reply portfolio/src/main/webapp/map.js:17]: My bad, will bring it back!

File: portfolio/src/main/webapp/map.js (hunk 3 above)
Comment (apluscs) [reply portfolio/src/main/webapp/map.js:18]: Right now, no. ln the future, maybe?

**Pull request merged**